sklearn/cross_decomposition/_pls.py

```diff
@@ -584,6 +584,18 @@ class PLSRegression(_PLS):
     y_rotations_ : array, [q, n_components]
         Y block to latents rotations.
 
+    x_mean_ : array, [p]
+        X mean for each predictor.
+
+    y_mean_ : array, [q]
+        Y mean for each response variable.
+
+    x_std_ : array, [p]
+        X standard deviation for each predictor.
+
+    y_std_ : array, [q]
+        Y standard deviation for each response variable.
+
     coef_ : array, [p, q]
         The coefficients of the linear model: ``Y = X coef_ + Err``
 
```

Comment:
for the `n_iter_` under we will have something:

```
n_iter_ : ndarray of shape (n_components,)
```

Reply:
Thank you very much. I fixed this as same as the other parameters.